src/test/java/apoc/spatial/GeocodeTest.java

```diff
@@ -151,7 +151,7 @@ private void testGeocodeAddress(Map map, String provider) {
                 }
                 testCallCount(db, "CALL apoc.spatial.geocode({url},0)",
                         map("url", map.get("address").toString()),
-                        (int) ((Map) map.get("count")).get(provider));
+                        Integer.parseInt(((Map) map.get("count")).get(provider).toString()));
```

Comment:
use Number.intValue()

Reply:
We restored the previous version the value of `((Map) map.get("count")).get(provider))` is aready and integer.